docs/source/user_guide/get_started/examples/environ_calculation.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+
+from aiida import orm
+from aiida.engine import run, submit
+import numpy as np
+from ase import Atoms
+from ase import units
+"""
+Calculation that recreates the environ calculation of
+CO on Pt(111) with a unit charge
+
+More details about the solvation models and potential corrections
+can be found here:
+
+O. Andreussi, I. Dabo and N. Marzari, J. Chem. Phys. 136,064102 (2012).
+I. Dabo et al. Phys.Rev. B 77, 115139 (2008)
+
+This system is taken directly from Example04 in the
+examples folder from Environ
+"""
+
+
+def calculator():
```

Comment:
Why did you make this and the `environ_calculator` a function if all it does is return a static dictionary? I think it might be simpler if to literally just include the dictionary in the main method where you assign it directly to the building

Reply:
Yup good point, has been corrected.